src/Propel/Generator/Manager/DataDictionaryExportManager.php

```diff
@@ -0,0 +1,263 @@
+<?php
+
+/**
+ * MIT License. This file is part of the Propel package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Propel\Generator\Manager;
+
+use Propel\Generator\Model\Column;
+use Propel\Generator\Model\Database;
+use Propel\Generator\Model\Table;
+
+/**
+ * Manager for Markdown Data Dictionary
+ *
+ * @author Charles Crossan <crossan007@gmail.com>
+ */
+class DataDictionaryExportManager extends AbstractManager
+{
+    public const DICTIONARYCOLUMNS = ['Column Name', 'PHP Name', 'Type', 'Length', 'PK', 'NN', 'UQ', 'AI', 'FK', 'Default', 'Description'];
```

Comment:
I wonder if those should be two words: DICTIONARY_COLUMNS

Reply:
Good point, that wasn't named very well. Changed it to `COLUMN_TABLE_HEADERS`